rados/ceph_buffer_list.go

```diff
@@ -0,0 +1,53 @@
+//go:build ceph_preview
+// +build ceph_preview
```

Comment:
Did you intentionally add `ceph_preview` build tag in this file?

I should have made it clear that you add preview tags only to those new files where APIs are added(and corresponding test files). I feel like the contents from this file are mostly helper functions and definitions. Also make sure that you run `make api-update` once preview tags are removed.

Reply:
Good thought.

You could probably make use of already existing wrappers from _internal/cutil/buffergroup.c_. I'm not sure though.